barman/clients/cloud_backup_delete.py

```diff
@@ -177,7 +184,7 @@ def _delete_backup(
             cloud_interface.delete_objects([backup_info_path])
         except Exception as exc:
             logging.error("Could not delete backup %s: %s", backup_id, force_str(exc))
-            raise SystemExit(2)
+            raise OperationErrorExit()
```

Comment:
exit value changes from 2 to 1. Not sure if it is on purpose.

Reply:
This was deliberate since the error is something which occurred while attempting to perform the deletion (but we don't know anything more than that).